website/content/api-docs/secret/identity/oidc-provider.mdx

```diff
@@ -240,4 +240,260 @@ $ curl \
+      "kid": "ee7c0920-fdb9-5c1a-9c69-6dab710d1a09",
+      "alg": "RS256",
+      "n": "zdFjUV9lBw5nQPvTtwH-gzKgRG7iepvYbFoc2hNB0-inJL25oh-mvNW3GS8jPY5XHLsiWa_1TKKE99JrKQgane2C96soFeOvR7SozbCeH8_FpZelH1Pym1NV038j05Vp87uB9FeKPsy1PNOLPTs_Fp42JIAenly7ojYwPp1s61p9V0U9rOhtldY7GkXHLN9s8v3aJjxqrTS3Puhs9MFS7EgRrEDAc69uiLXCoYXKygjXddvJi6j446XxnO2eTRMGl1f2t04s_vDgVnFQgjQSKYWPbOMhf2slkeR47fqE3qqUDzINxauqMbkW-PlLP9IN0crR2uC07cG2os4RxN4YHw",
+      "e": "AQAB"
+    },
+    {
+      "use": "sig",
+      "kty": "RSA",
+      "kid": "6e468221-b7c2-9d2d-744d-33b7ae0357cb",
+      "alg": "RS256",
+      "n": "rMaucILJKiFg_lkCE8ZEV_8jiYdaVDjKkc-8XPBW8S34wIRl1EbsgCYfMHtJnIJ_3eUgOVorW5KVeN9C8W16LR3lhqRWS9y4qlt0AcWpOvsmxr5q5dS_QqgCjeftCKwJzUsMi5bMW8wKjRZdd-qLz6X1rVSZWX82G0So8nRBg9d3MNJbKcdIJrRbrxWkm8U9xMqRouzbyQ2Hsp2rRVgGh7yjEA6daI5Ao8UsPdBmlCM9oKZ1_Kje5JTfZKeHlT-58vn_ylCjMVlapLuUsDN6He2kPVyOzGbie297VOfjmB7QX0ah1f7Ni1UJFJYHrVK9wMfCLTltSFZBcQ9--FlVdQ",
+      "e": "AQAB"
+    }
+  ]}
+```
+
+## Authorization Endpoint
+
+Provides the [Authorization Endpoint](https://openid.net/specs/openid-connect-core-1_0.html#AuthorizationEndpoint)
+for an OIDC provider. This allows OIDC clients to request an authorization code
+to be used for the [Authorization Code Flow](https://openid.net/specs/openid-connect-core-1_0.html#CodeFlowAuth).
+
+| Method      | Path                                      |
+| :---------- | :---------------------------------------- |
+| `GET/POST`  | `/identity/oidc/provider/:name/authorize` |
+
+### Parameters
+
+- `name` `(string: <required>)` - The name of the provider. This parameter is specified as part of the URL.
+
+- `scope` `(string: <required>)` - A space-delimited list of scopes to be requested. The `openid` scope is required.
+
+- `response_type` `(string: <required>)` - The OIDC authentication flow to be used. The following response types are supported: `code`.
+
+- `client_id` `(string: <required>)` - The ID of the requesting client.
+
+- `redirect_uri` `(string: <required>)` - The redirection URI to which the response will be sent.
+
+- `state` `(string: <required>)` - A value used to maintain state between the authentication request and client.
+
+- `nonce` `(string: <required>)` - A value that is returned in the ID token nonce claim. It is used to mitigate replay attacks.
+
+### Sample Request
+
+```shell-session
+$ curl \
+    --request GET \
+    --header "X-Vault-Token: ..." \
+    -G \
+    -d "response_type=code" \
+    -d "client_id=$CLIENT_ID" \
+    -d "state=af0ifjsldkj" \
+    -d "nonce=abcdefghijk" \
+    --data-urlencode "scope=openid" \
+    --data-urlencode "redirect_uri=http://127.0.0.1:8251/callback" \
+    http://127.0.0.1:8200/v1/identity/oidc/provider/test-provider/authorize
+```
+
+### Sample Response
+
+```json
+{
+  "code": "BDSc9kVYljxND93YpveBuJtSvguM3AWe",
+  "state": "af0ifjsldkj"
+}
+```
+
+## Token Endpoint
+
+Provides the [token endpoint](https://openid.net/specs/openid-connect-core-1_0.html#TokenEndpoint)
```

Comment:
Suggest to capitalize like "Token Endpoint" similar to "Authorization Endpoint" above. It's a small detail, but I like that it matches up with how it's capitalized in the spec.